Lib/test/test_code.py

```diff
@@ -337,6 +337,37 @@ def func():
         new_code = code = func.__code__.replace(co_linetable=b'')
         self.assertEqual(list(new_code.co_lines()), [])
 
+    def test_invalid_bytecode(self):
+        def foo(): pass
+        co = foo.__code__
+        CodeType = type(co)
+
+        new_co = CodeType(
+            co.co_argcount,
+            co.co_posonlyargcount,
+            co.co_kwonlyargcount,
+            co.co_nlocals,
+            co.co_stacksize,
+            co.co_flags,
+            b'\xee\x00d\x00S\x00',
+            co.co_consts,
+            co.co_names,
+            co.co_varnames,
+            co.co_filename,
+            co.co_name,
+            co.co_qualname,
+            co.co_firstlineno,
+            co.co_linetable,
+            co.co_exceptiontable,
+            co.co_freevars,
+            co.co_cellvars)
+
+        foo.__code__ = new_co
+        with self.assertRaises(SystemError) as se:
+            foo()
+        self.assertIn("unknown opcode '238'", str(se.exception))
```

Comment:
Would it be possible to check the filename, and maybe even the line number? Try values from ``foo.__code__``?

* ``foo.__code__.co_filename``
* ``foo.__code__.co_firstlineno``